definitions/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "definitions",
+  "version": "1.0.0",
+  "scripts": {
+    "test": "ts-node ./build.ts"
+  },
+  "dependencies": {
+    "tapable": "../"
```

Comment:
This is not yet used... I'm not exactly sure how I would like to actually test the type definitions for comprehensive coverage.